pkg/operator/ceph/csi/controller.go

```diff
@@ -140,16 +150,36 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 
 		return reconcile.Result{}, nil
 	} else {
```

Comment:
nit: this else block isn't necessary since the `if` code path always returns, which helps the later block have lesser indentation.
```suggestion
	} 
```